src/Base/XMLTools.h

```diff
@@ -174,6 +174,12 @@ inline XStr::~XStr()
     XERCES_CPP_NAMESPACE_QUALIFIER XMLString::release(&fUnicodeForm);
 }
 
+#define XStrLiteral(literal)                                                                       \

```

Comment:
```suggestion
// Uses the compiler to create a cache of transcoded string literals so that each subsequent call
// can re-use the data from the lamda's initial creation. Permits the same usage as 
// XStr("literal").unicodeForm()
#define XStrLiteral(literal)                                                                       \
```